packages/components/src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { useStyles } from "../../hooks/useStyles";
+import { Color, EDSStyleSheet } from "../../styling";
+import { View } from "react-native";
+import { Typography } from "../Typography";
+import { PressableHighlight } from "../PressableHighlight";
+import { Icon, IconName } from "../Icon";
+import { Button } from "../Button";
+
+export type ChipProps = {
+    /**
+     * Text to render in the chip.
+     */
+    title: string;
+    /**
+     * The chip variant. Determines the visual expression of the chip.
+     */
+    variant?: "default" | "error" | "active";
+    /**
+     * Whether or not the chip should respond to touch events and render its disabled state.
+     */
+    disabled?: boolean;
+    /**
+     * Callback method invoked when the user presses the chip. If not defined, the chip is not pressable.
+     */
+    onPress?: () => void;
+    /**
+     * Callback method invoked when the user presses the delete button on the right side of the chip. If not defined, the chip does not render the delete button.
+     */
+    onDelete?: () => void;
+    /**
+     * Optional icon to render on the left side of the chip.
+     */
+    iconName?: IconName;
+};
+
+export const ICON_SIZE = 16;
```

Comment:
I dont see this export being used anywhere else in the monorepo. Any reason to export it?

Reply:
Shiee, good point

Reply:
fixed in d2d0035